.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Create a bug report.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**What happened?**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior: 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+ - If applicable, add screenshots to help explain your problem.
```

Comment:
Unless they are using the `crm graph` command, screenshots are not preferred. This is not a very graphical library. I want logs, not image files I can't copy text out of.

If we could include the triple-back-tic block section for them, that might be nice to include in the template.